File: data/data.code.java.server/AckTestJsOk.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckTestJsOk
{
	private long u64;
	private long i64;


	public byte[] encode()
	{
		Packet packet = new Packet();
		packet.writeLong(u64);
		packet.writeLong(i64);
		return packet.encode((short)Msg.P_ACK_TEST_JS_OK);
	}



	public void setU64(long u64)
	{
		this.u64 = u64;
	}

	public void setI64(long i64)
	{
		this.i64 = i64;
	}

}

File: examples/golang/wsserver/proto/ack_test_js_ok.go
package proto

import (
	"packet"
)

type AckTestJsOk struct {
	u64                      uint64
	i64                      int64
}

func (this *AckTestJsOk) Encode() []byte {
	pack := packet.NewWriteBuff(64)

	pack.WriteUint64(this.u64)
	pack.WriteInt64(this.i64)

	return pack.Encode(P_ACK_TEST_JS_OK)
}

func (this *AckTestJsOk) SetU64(u64 uint64) {
	this.u64 = u64
}

func (this *AckTestJsOk) SetI64(i64 int64) {
	this.i64 = i64
}
